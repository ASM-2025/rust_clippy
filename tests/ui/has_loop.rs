#![allow(unused)]
#![warn(clippy::has_loop)]

fn main() {
    // test code goes here
    loop {
        println!("hello, world")
    }
}
