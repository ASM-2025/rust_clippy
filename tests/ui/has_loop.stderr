error: Loop detected in function "main"
  --> $DIR/has_loop.rs:6:5
   |
LL | /     loop {
LL | |         println!("hello, world")
LL | |     }
   | |_____^
   |
   = note: `-D clippy::has-loop` implied by `-D warnings`

error: aborting due to previous error

